2048/2048/src/main/res/layout/activity_finish_game.xml

```diff
@@ -58,6 +58,22 @@
         app:layout_constraintTop_toBottomOf="@+id/animation_description_text"
         />
 
+    <TextView
+        android:id="@+id/third_message"
```

Comment:
you should give a better ID to this `TextView`

Reply:
Replaced it with extra_info? is that good enough?